packages/lib-classifier/src/components/Classifier/components/ModalTutorial/ModalTutorial.js

```diff
@@ -23,21 +23,26 @@ function storeMapper (stores) {
 @inject(storeMapper)
 @observer
 class ModalTutorial extends React.Component {
+  onClose () {
+    const { setModalVisibility } = this.props
+    setModalVisibility(false)
+  }
+
   render () {
-    const { loadingState, showModal, setModalVisibility, tutorial } = this.props
+    const { loadingState, showModal, tutorial } = this.props
     if (loadingState === asyncStates.success && tutorial) {
       return (
         <Modal
           {...this.props}
           active={showModal}
-          closeFn={() => { setModalVisibility(false) }}
+          closeFn={this.onClose.bind(this)}
```

Comment:
Minor point, but passing on some advice from @rogerhutchings: this might create a new bound function on each call to `render` so I've been switching back to binding event handlers in the component constructor.

Reply:
Oh, good point. I thought this was the preferred way according to the React docs, but I just re-read it and it does point out the potential performance issue here. Binding inline is only necessary to pass arguments. Thanks.